code/game/objects/items/weapons/twohanded.dm

```diff
@@ -272,3 +272,36 @@
 
 	force = 16
 	throwforce = 25
+
+/obj/item/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	icon_state = "binoculars"
+	item_state = "syringe_kit"
```

Comment:
I have no idea how you would make a sprite for holding the binoculars up like that.